mongodbatlas/data_source_mongodbatlas_project.go

```diff
@@ -175,6 +209,12 @@ func dataSourceMongoDBAtlasProjectRead(ctx context.Context, d *schema.ResourceDa
 		}
 		log.Println("[WARN] `api_keys` will be empty because the user has no permissions to read the api keys endpoint")
 	}
+
+	limits, _, err := connV2.ProjectsApi.ListProjectLimits(ctx, project.ID).Execute()
+	if err != nil {
+		return diag.Errorf("error getting project's limits (%s): %s", projectID, err)
```

Comment:
more info here https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w
```suggestion
		return diag.Errorf("error getting project's limits (%s): %w", projectID, err)
```

Reply:
closing as Diagnostics currently does not support error-wrapping directive